docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -1,20 +1,69 @@
 .. _arch_overview_load_balancing:
 
-Load balancing
+Load Balancing
 ==============
 
+What is Load Balancing?
+-----------------------
+
+Load balancing is a way of distributing traffic between multiple hosts within a single upstream cluster in order to effectively make use of available resources. There are many different ways
+of accomplishing this, so Envoy provides several different load balancing strategies.
+At a high level, we can break these strategies into two categories: global
+load balancing and distributed load balancing.
+
+.. _arch_overview_load_balancing_global_lb:
+
+Global Load Balancing
+---------------------
+
+Global load balancing refers to having a single, global authority that decides how load should
+be distributed between hosts. For Envoy, this would be done by the control plane, which is able
+to adjust the load applied to individual endpoints by specifying various parameters, such as
+priority, locality weight, endpoint weight and endpoint health.
+
+A simple example would be to have the control plane assign hosts to different
+:ref:`priorities <arch_overview_load_balancing_priority_levels>` based on network topology
+to ensure that hosts that require fewer network hops are preferred. This is similar to
+zone-aware routing, but is handled by the control plane instead of by Envoy. A benefit of doing
+it in the control plane is that it gets around some of the
+:ref:`limitations <arch_overview_load_balancing_zone_aware_routing_preconditions>` of zone aware routing.
+
+A more complicated setup could have resource usage being reported to the control plane, allowing
+it to adjust the weight of endpoints or :ref:`localities <arch_overview_load_balancing_locality_weighted_lb>`
+to account for the current resource usage, attempting to route new requests to idle hosts over busy ones.
+
+.. _arch_overview_load_balancing_distributed_lb:
+
+Distributed Load Balancing
```

Comment:
I would probably introduce distributed load balancing first since IMO it's simpler to understand.